File: Java Codewars/8kyu/Calculate average.java
/*
Description

Write a function which calculates the average of the numbers in a given list.

Note: Empty arrays should return 0.
*/

public class Kata {

  public static double find_average(int[] array) {
    int length = array.length;
    double sum = 0;
    for (int number : array) {
      sum += number;
    }
    return sum / length;
  }
}

File: PHP Codewars/8kyu/Calculate average.php
<?php 
/*
Description

Write a function which calculates the average of the numbers in a given array.

Note: Empty arrays should return 0.
*/

function find_average($array): float {
  if (count($array) === 0) {
    return 0;
  }
  $sum = array_sum($array);
  return $sum / count($array);
}
?>